<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QDialog" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="feed_btn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Feed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_view_image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>View Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_red_mark">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="profile_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>35</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>50</y>
     <width>351</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_import">
        <property name="text">
         <string>Import</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_statistics">
        <property name="text">
         <string>Get Statistics</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget_profile"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="notifications_bttn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Notifications</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <zorder>feed_btn</zorder>
  <zorder>pushButton_view_image</zorder>
  <zorder>profile_btn</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>notifications_bttn</zorder>
  <zorder>search_btn</zorder>
  <zorder>logout_btn</zorder>
  <zorder>label_red_mark</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
